src/common/services/api/order.service.js

```diff
@@ -372,8 +372,8 @@ class Order {
   }
 
   storeRadioStationData (radioStationData) {
-    this.sessionStorage.setItem('radioStationName', radioStationData.Description)
-    this.sessionStorage.setItem('radioStationCallLetters', radioStationData.MediaId)
+    this.sessionStorage.setItem('radioStationName', Object.values(radioStationData)[0])
+    this.sessionStorage.setItem('radioStationCallLetters', Object.keys(radioStationData)[0])
```

Comment:
I assume we know the names of these properties or they change?

Reply:
The names of the properties change based on zip code/API response.